test/util.spec.js

```diff
@@ -2198,4 +2198,157 @@ describe('UTIL', function() {
         });
     });
 
+    describe('#getMediaTypeObject', function() {
+        var nativeConfiguration, sizes, currentSlot;
+
+        beforeEach(function(done) {
+            nativeConfiguration =  {
+                kgp:"_DIV_", // Or it Could be _AU_
+                klm:{
+                    "DIV_1":{
+                        "nativeOnly": false,
+                        config: {
+                            image: {
+                                required: true,
+                                sizes: [150, 50]
+                            },
+                            title: {
+                                required: true,
+                                len: 80
+                            },
+                            sponsoredBy: {
+                                required: true
+                            },
+                            body: {
+                                required: true
+                            }
+                        }
+                    },
+                    "DIV_2":{
+                        "nativeOnly": true,
+                        config: {
+                            image: {
+                                required: true,
+                                sizes: [150, 50]
+                            },
+                            title: {
+                                required: true,
+                                len: 80
+                            },
+                            sponsoredBy: {
+                                required: true
+                            },
+                            body: {
+                                required: true
+                            }
+                        }
+                    }
+                }
+            };
+            sizes = [[300,250]];
+            currentSlot = { 
```

Comment:
suggestion: create a common method to create a slotObject / gptObject for unit-testing, this will avoid duplication of test-object creation code.

Reply:
Sure will refactor